src/qt/rpcconsole.cpp

```diff
@@ -866,7 +866,11 @@ void RPCConsole::clear(bool keep_prompt)
     }
 
     // Set default style sheet
+#ifdef __APPLE__
```

Comment:
```suggestion
#ifdef Q_OS_MAC
```

as we use it in our Qt code (sorry for not mentioning this earlier).